terraform/spec/dependabot/terraform/file_parser_spec.rb

```diff
@@ -33,7 +55,14 @@
       it "raises an error" do
         expect { subject }.to raise_error(Dependabot::DependencyFileNotParseable) do |boom|
           expect(boom.file_path).to eq("/main.tf")
-          expect(boom.message).to eq("unable to parse HCL: object expected closing RBRACE got: EOF")
+          if PackageManagerHelper.use_terraform_hcl2?
+            expect(boom.message).to eq(
+              "Failed to convert file: parse config: [:18,1-1: Argument or block definition required; " \
+              "An argument or block definition is required here.]"
+            )
+          else
+            expect(boom.message).to eq("unable to parse HCL: object expected closing RBRACE got: EOF")
+          end
```

Comment:
I'd prefer to make these two separate tests, using the `:hcl1_only` tag, that makes it easier to remove. Will do this now